src/abis/ERC20Guild.json

```diff
@@ -0,0 +1,1031 @@
+{

```

Comment:
Do we need ABIs duplicated here as well as in contracts/ ?

Reply:
I'm not sure what the convention is since @mprasanjith created this folder. I can remove this file specifically and keep it in contracts/